geode-core/src/main/java/org/apache/geode/security/SecurityManager.java

```diff
@@ -74,8 +74,11 @@ default void init(Properties securityProps) {}
    * @param principal The principal that's requesting the permission
    * @param permission The permission requested
    * @return true if authorized, false if not
+   *
+   * @throw AuthenticationExpiredException
```

Comment:
The tag is `@throws`. Explain when the exception is thrown as part of the detail.

```
@throws AuthenticationExpiredException if the principal has expired.
```